src/CustomParser/Advanced.elm

```diff
@@ -1058,6 +1058,43 @@ mapChompedString func (Parser parse) =
         )
 
 
+chompIfFollowedBy : (Char -> Bool) -> x -> Parser x a -> Parser x a
+chompIfFollowedBy isGood expecting (Parser parseNext) =
```

Comment:
Took me a bit of time to understand this one. I read it as "chomp, if followed by" so I thought it was doing look ahead but couldn't understand where :joy: 
(Leave it as is, it's fine, unless you have a better name in mind)